Kickstarter-iOS/Views/Controllers/SettingsNotificationsViewController.swift

```diff
@@ -174,6 +197,28 @@ extension SettingsNotificationsViewController: UITableViewDelegate {
     return headerView
   }
 
+  func tableView(_ tableView: UITableView, viewForFooterInSection section: Int) -> UIView? {
+    guard let sectionType = dataSource.sectionType(
+      section: section,
+      user: AppEnvironment.current.currentUser
+    ) else {
+      return nil
+    }
+
+    if sectionType.hasDescriptionFooter {
```

Comment:
You can remove the if statement below if you move `sectionType.hasDescriptionFooter ` to the guard above